scripts/RepoInfo.json

```diff
@@ -15,5 +15,6 @@
   "https://raw.githubusercontent.com/bl-sdk/former-default-mods/master/modinfo.json",
   "https://raw.githubusercontent.com/tominator1pl/bl-sdk-mods/main/ModInfo.json",
   "https://raw.githubusercontent.com/Cae-l/bl-sdk-mods/master/ModInfo.json",
-  "https://gist.githubusercontent.com/mopioid/d0811b04ff350c117a0f8bd1d2b3ad88/raw/e5c091f74ed0ff4f7d8b182cd1ce9d0911f1ba9c/modinfo.json"
+  "https://gist.githubusercontent.com/mopioid/d0811b04ff350c117a0f8bd1d2b3ad88/raw/e5c091f74ed0ff4f7d8b182cd1ce9d0911f1ba9c/modinfo.json",
+  "https://raw.githubusercontent.com/Zazk0u/bl-sdk-mods/main/modinfo.json",
```

Comment:
Can't have a trailing comma here, that's what's causing the build to fail.

Also noticed in your modinfo, the versions don't line up properly, latest is 1.1, but versions only has 1.0

Reply:
Alright thanks